conf/messages.cy

```diff
@@ -271,6 +287,26 @@ error.timeOfArrival.hour.invalid-digits=Mae’n rhaid i’r awr cyrraedd gynnwys
 error.timeOfArrival.minutes.invalid-digits=Mae’n rhaid i’r munudau cyrraedd gynnwys rhifau yn unig
 error.timeOfArrival.hour.invalid-value=Mae’n rhaid i’r oriau fod rhwng 0 a 23
 error.timeOfArrival.minutes.invalid-value=Mae’n rhaid i’r munudau fod rhwng 00 a 59
+
+error.dateOfDeparture.all.required=Nodwch y dyddiad ymadael
+error.dateOfDeparture.year.required=Mae’n rhaid i’r dyddiad ymadael gynnwys blwyddyn
+error.dateOfDeparture.month.required=Mae’n rhaid i’r dyddiad ymadael gynnwys mis
+error.dateOfDeparture.day.required=Mae’n rhaid i’r dyddiad ymadael gynnwys diwrnod
+error.dateOfDeparture.all.invalid-value=Mae’n rhaid i’r dyddiad ymadael fod yn ddyddiad go iawn
+error.dateOfDeparture.year.invalid-value=Mae’n rhaid i’r dyddiad ymadael fod yn ddyddiad go iawn
+error.dateOfDeparture.month.invalid-value=Mae’n rhaid i’r dyddiad ymadael fod yn ddyddiad go iawn
+error.dateOfDeparture.day.invalid-value=Mae’n rhaid i’r dyddiad ymadael fod yn ddyddiad go iawn
+error.dateOfDeparture.year.invalid-digits=Mae’n rhaid i’r flwyddyn ymadael gynnwys rhifau yn unig
+error.dateOfDeparture.month.invalid-digits=Mae’n rhaid i’r mis ymadael gynnwys rhifau yn unig
+error.dateOfDeparture.day.invalid-digits=Mae’n rhaid i’r diwrnod ymadael gynnwys rhifau yn unig
+error.dateOfDeparture.all.invalid-value-range=Mae’n rhaid i’r dyddiad ymadael fod rhwng {0} a {1}
+error.dateOfDeparture.all.invalid-value-before-entry-date=Mae’n rhaid i’r dyddiad ymadael fod rhwng {0} (dyddiad mynediad) a {1}
+error.timeOfDeparture.all.required=Nodwch yr amser ymadael amcangyfrifedig
+error.timeOfDeparture.hour.required=Mae’n rhaid i’r amser ymadael gynnwys awr
+error.timeOfDeparture.minutes.required=Mae’n rhaid i’r amser ymadael gynnwys munudau
+error.dateOfArrival.all.invalid-value-range=Mae’n rhaid i’r dyddiad cyrraedd fod rhwng {0} a {1}
+error.dateOfArrival.all.invalid-value-before-entry-date=Mae’n rhaid i’r dyddiad cyrraedd fod rhwng {0} (dyddiad mynediad) a {1}
```

Comment:
Two errors missing here, Irene has confirmed we can use: 

error.timeOfDeparture.hour.invalid-digits
=> Mae’n rhaid i’r awr ymadael gynnwys rhifau yn unig

error.timeOfDeparture.minutes.invalid-digits
=> Mae’n rhaid i’r munudau ymadael gynnwys rhifau yn unig